src/null-safety/index.md

```diff
@@ -5,37 +5,36 @@ description: Information about Dart's null safety feature
 
 The Dart language comes with sound null safety. 
 
-In Dart 3, you always get null safety. In Dart 2.x,
```

Comment:
This page is a historical document for the introduction of null safety, not documentation of the null-safe type system, consider keeping this, especially since it's just after Dart 3 released.

Reply:
I think the intro part of this page should be brought up into the variables page above.

Reply:
Done.